jar_test.go

```diff
@@ -1964,6 +1964,63 @@ func TestRemoveCookies(t *testing.T) {
 	}
 }
 
+func TestRemoveAll(t *testing.T) {
+	jar := newTestJar("")
+	google := mustParseURL("https://www.google.com")
+	apple := mustParseURL("https://www.apple.com")
+	cookies := []*http.Cookie{
```

Comment:
Let's have some tests with cookies set for sub-paths, please, and some
that remove cookies with a URL with a path set.

For example, RemoveAll("https://foo.com/bar/") should not remove cookies
set for https://foo.com/.